lab-nicole/__test__/auth-route.test.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const request = require('superagent');
+const User = require('../model/user.js');
+const serverToggle = require('../lib/server-toggle.js');
+const server = require('../server.js');
+
+require('jest');
+
+const url = 'http://localhost:3000';
+
+const exampleUser = {
+  username: 'exampleuser',
+  password: '1234',
+  email: 'exampleuser@test.com',
+};
+
+describe('Auth Routes', function() {
+  beforeAll( done => {
+    serverToggle.serverOn(server, done);
+  });
+
+  afterAll( done => {
+    serverToggle.serverOff(server, done);
+  });
+
+  describe('POST: /api/signup', function() {
+    afterEach( done => {
+      User.remove({})
+        .then( () => done())
```

Comment:
Why do you have a space after the opening parenthesis but not before the closing one? Why do you have the space here but not everywhere? Shoot for consistency